src/Mailhog_Command.php

```diff
@@ -43,6 +43,7 @@ public function up( $args, $assoc_args ) {
 		}
 		EE::docker()::docker_compose_up( $this->site_data->site_fs_path, [ 'mailhog' ] );
 		EE::exec( "docker-compose exec postfix postconf -e 'relayhost = mailhog:1025'" );
+		EE::exec( 'docker-compose restart postfix' );
```

Comment:
https://github.com/EasyEngine/mailhog-command/blob/d8e400ba552e127a8d666a74a6b5b1a338513df4/src/Mailhog_Command.php#L71

Also need to add the restart command after removing mailhog